internal/snap_api/snap_api_test.go

```diff
@@ -44,6 +44,74 @@ __commonJS["./foo.js"] = function(exports, module2, __filename, __dirname, requi
 	)
 }
 
+func TestTypeScriptImportingDefaultModule(t *testing.T) {
```

Comment:
This is a test of `import` functionality. It was broken without the corresponding change.